sample/src/main/kotlin/com/slack/circuit/sample/petlist/PetList.kt

```diff
@@ -229,8 +231,9 @@ private fun PetListGrid(
   animals: List<PetListAnimal>,
   eventSink: (PetListScreen.Event) -> Unit
 ) {
+  val isLandscape = LocalConfiguration.current.orientation == Configuration.ORIENTATION_LANDSCAPE
   LazyVerticalGrid(
-    columns = GridCells.Fixed(2),
+    columns = GridCells.Fixed(if (isLandscape) 3 else 2),
```

Comment:
maybe we can make this numbers constants later in cleanup? idk

Reply:
We could, but I also feel like trivial cases like this are sorta ok. It's obvious reading it what they're for, vs say something like a function with several number params that are all unlabeled